### PR TITLE
Reader: add combined card post placeholder

### LINKED ISSUE #12072: Reader: add loading placeholder for combined card
We should add a loading placeholder for combined card posts.

### PR DESCRIPTION
**Pull request by bluefuton**
This PR adds a loading placeholder for posts in a Reader combined card.

Loading extra posts currently looks a bit odd when clicking the new post pill (e.g. "2 new posts") - the visit link is shown and then the remaining post content arrives a moment later. The placeholder should fix that.

Fixes #12072.

screen shot 2017-03-14 at 17 43 22

### To test

If you want to manually set the loading state, you can set `isLoading` to `true` in reader-combined-card/placeholders/post.jsx.

The combined card devdocs page is http://calypso.localhost:3000/devdocs/blocks/reader-combined-card.

### CODE REVIEW

**Comment by fraying** [comment]
I see the placeholders!

screen shot 2017-03-14 at 10 49 38 am

They look great!

But I never see them resolve (turn into the real posts). Is that on purpose?

**Comment by bluefuton** [comment]
Apologies @fraying - I still had `isLoading` forced to `true` for testing :) Fixed now.

File: client/blocks/reader-combined-card/post.jsx [hunk 1]
```diff
@@ -71,8 +72,13 @@ class ReaderCombinedCardPost extends React.Component {
 
 	render() {
 		const { post, streamUrl, isDiscover, isSelected } = this.props;
-		const hasAuthorName = has( post, 'author.name' );
+		const isLoading = ! post || post._state === 'pending';
```
Comment (blowery) [inline client/blocks/reader-combined-card/post.jsx]: could also be `minimal` i think

Comment (bluefuton) [inline client/blocks/reader-combined-card/post.jsx]: I wondered about that - we only check for `pending` in full post view. I'll update both.

**Comment by bluefuton** [comment]
@blowery now checking for post state `minimal` in both combined card and full post view.

**Pull request merged**